<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2CK62EB" failures="1" tests="1" name="Surefire test" time="14.846" errors="0" timestamp="2021-10-29T17:02:47 EDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="14.484">
    <failure type="java.lang.StringIndexOutOfBoundsException" message="begin 0, end 21, length 0">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: begin 0, end 21, length 0
at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3734)
at java.base/java.lang.String.substring(String.java:1903)
at pageObject.SignUp_page.checkSuccessMsgWithoutId(SignUp_page.java:154)
at stepDefinition.signUp_steps.iShouldSeeThankYouForSignUpMessage(signUp_steps.java:47)
at âœ½.I should see Thank you for sign up message(file:///C:/Users/syeda/Documents/TalentTEK/git/Cucumber_BDD/src/test/frameWork/features/signUp.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Surefire test -->
